provider-ci/src/steps.ts

```diff
@@ -197,13 +197,13 @@ export function InstallPulumiCtl(): Step {
   };
 }
 
-export function InstallSchemaChecker(): Step {
+export function InstallSchemaChecker(preferredSchemaChecker: string): Step {
```

Comment:
It might be nice to make the valid set of inputs explicit here:

```suggestion
type PreferredSchemaChecker = "mikhailshilkov/schema-tools" | "pulumi/schema-tools"
export function InstallSchemaChecker(preferredSchemaChecker: PreferredSchemaChecker): Step {
```

This will mean that the compiler will tell you if we update the allowed config values but forget to handle the addtional cases here.